src/adapter/src/coord/peek.rs

```diff
@@ -88,7 +89,7 @@ where
                     *ctx.as_mut() += 1;
                     fmt_text_constant_rows(
                         f,
-                        rows.iter().map(|(row, _, diff)| (row, diff)),
+                        rows.iter().map(|(row, diff)| (row, diff)),
```

Comment:
Is just `rows.iter()` sufficient here?

Reply:
Doing that produces the following error:
```
error[E0271]: expected `std::slice::Iter<'_, (mz_repr::Row, i64)>` to be an iterator that yields `(&mz_repr::Row, &i64)`, but it yields `&(mz_repr::Row, i64)`
   --> src/adapter/src/coord/peek.rs:90:47
    |
90  |                     fmt_text_constant_rows(f, rows.iter(), ctx.as_mut())?;
    |                     ----------------------    ^^^^^^^^^^^ expected tuple, found reference
    |                     |
    |                     required by a bound introduced by this call
    |
    = note:  expected tuple `(&mz_repr::Row, &i64)`
            found reference `&(mz_repr::Row, i64)`
note: required by a bound in `fmt_text_constant_rows`
   --> /home/joe/materialize/src/repr/src/explain_new/mod.rs:661:17
    |
661 |     I: Iterator<Item = (&'a Row, &'a crate::Diff)>,
    |                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ required by this bound in `fmt_text_constant_rows`
```
I'm not really sure of a cleaner way to fix it.